### PR TITLE
Add navbarlogo

### PR DESCRIPTION
**Pull request by Dindihub**
Addressing issue #3 

@chalin I'd appreciate your review on this.

### CODE REVIEW

**Comment by chalin** [comment]
> @chalin I'd appreciate your review on this.

Gladly, but the build is failing, so we can't see the preview. I think that it's because you updated Docsy. Maybe wait for the next official Docsy release before updating, or submit the update through a separate PR.

**Comment by chalin** [comment]
For https://app.netlify.com/sites/addtuflogo/deploys/66b28e380699800007468302, try clearing the cache and triggering a rebuild (within Netlify). That might resolve the issue with the Docsy theme repo

File: hugo.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # cSpell:ignore docsy github goldmark linkify netlify
 
 baseURL: https://cncf-docsy-starter.netlify.app/ # CUSTOMIZE
-title: Docsy Starter # CUSTOMIZE
+title: # CUSTOMIZE
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
title: TUF
```

File: hugo.yaml [hunk 2]
```diff
@@ -37,7 +37,7 @@ languages:
     languageName: English
     languageCode: en-US
     params:
-      description: Docsy starter # CUSTOMIZE
+      description: TUF # CUSTOMIZE
```
Comment (chalin) [inline hugo.yaml]: Add the project description here.

**Comment by Dindihub** [comment]
> > @chalin I'd appreciate your review on this.
> 
> Gladly, but the build is failing, so we can't see the preview. I think that it's because you updated Docsy. Maybe wait for the next official Docsy release before updating, or submit the update through a separate PR.

Apologies @chalin. I realized too late that it wasn't building. Was to handle it today. Thanks for the suggestions 💯

**Comment by Dindihub** [comment]
@chalin I fixed the logo issue, please review. Thanks

File: assets/scss/_styles_project.scss [hunk 3]
```diff
@@ -1 +1,16 @@
...
+        padding: 0;
+      }
+      svg {
+        height: 48px;
+      }
+      .navbar-brand__name {
+        display: none;
+      }
+    }
+  }
```
Comment (chalin) [inline assets/scss/_styles_project.scss]:
```suggestion
  }

```

File: assets/scss/_styles_project.scss [hunk 4]
```diff
@@ -1 +1,16 @@
 @import 'td/code-dark';
+
+
+.td-navbar {
+    .navbar-brand {
+      @include media-breakpoint-up(md) {
+        padding: 0;
+      }
+      svg {
+        height: 48px;
```
Comment (chalin) [inline assets/scss/_styles_project.scss]:
```suggestion
        height: 38px;
```

File: hugo.yaml [hunk 5]
```diff
@@ -63,6 +63,7 @@ imaging:
   anchor: smart
 
 params:
+
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
```

File: hugo.yaml [hunk 6]
```diff
@@ -37,7 +37,7 @@ languages:
     languageName: English
     languageCode: en-US
     params:
-      description: Docsy starter # CUSTOMIZE
+      description: A Framework for securing software update systems # CUSTOMIZE
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
      description: A Framework for securing software update systems
```

File: assets/scss/_styles_project.scss [hunk 7]
```diff
@@ -1 +1,16 @@
 @import 'td/code-dark';
+
+
+.td-navbar {
+    .navbar-brand {
+      @include media-breakpoint-up(md) {
+        padding: 0;
+      }
```
Comment (chalin) [inline assets/scss/_styles_project.scss]: I think that this can be removed:

```suggestion
```

File: hugo.yaml [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # cSpell:ignore docsy github goldmark linkify netlify
 
-baseURL: https://cncf-docsy-starter.netlify.app/ # CUSTOMIZE
-title: Docsy Starter # CUSTOMIZE
+baseURL: https://theupdateframework.io/ # CUSTOMIZE
```
Comment (chalin) [inline hugo.yaml]:
```suggestion
baseURL: https://theupdateframework.io
```

File: assets/icons/tuf-horizontal-white.png
Comment (chalin) [inline assets/icons/tuf-horizontal-white.png:1]: I don't think that this file is being used. If so, let's drop it.

**Review by chalin — APPROVED** [review]
Well done! LGTM after you've addressed the inline comments and suggestions.

File: assets/icons/tuf-horizontal-white.png
Comment (chalin) [reply assets/icons/tuf-horizontal-white.png:1]: Hi. You marked this as resolved but I still see the `.png` file in the PR, and now in the repo. Was there a reason that you kept it?

**Pull request merged**